PythonWrapperGenerator/main.cpp

```diff
@@ -0,0 +1,18 @@
+#include <iostream>
+
+#include <QtCore/QDebug>
+#include <QtCore/QCoreApplication>
+
+#include "GenTool.h"
+#include "Config.h"
+
+int main(int argc, char *argv[])
+{
+	QCoreApplication a(argc, argv);
+	GenTool t;
+	t.setSubDirPath(Config::instance().envisionReadPath());
+	t.run();
+	qDebug() << "Done!";
```

Comment:
Don't use `qDebug` for actual output of the program. Isn't this the reason you included `iostream`?